### PR TITLE
[Sudo] feat: Add sudo spec

### PR DESCRIPTION
**Pull request by CastaChick**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Add spec for `sudo` 

**What is the current behavior? (You can also link to an open issue here)**
#176 

**What is the new behavior (if this is a feature change)?**
Show autocompletes after `sudo`

スクリーンショット 2021-05-08 17 22 19

**Additional info:**

### CODE REVIEW

**Comment by CastaChick** [comment]
This is my first contribution to Fig ✨
If I should fix something, I'm willing to fix it 😃

File: dev/sudo.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export const completion: Fig.Spec = {
+  name: "sudo",
+  description: "execute a command as the superuser or another user",
+  subcommands: [],
```
Comment (cstrnt) [inline dev/sudo.ts:4]: this can safely be removed 😊

File: dev/sudo.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  options: [
+    {
+      name: ["-g"],
```
Comment (cstrnt) [inline dev/sudo.ts:7]: I think it would be awesome if you could also add the long variant (--group) here

File: dev/sudo.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      description: "run command as the specified group name or ID",
+      args: {
+        name: "group",
+        description: "group name or ID",
+      },
+    },
+    {
+      name: ["-h"],
```
Comment (cstrnt) [inline dev/sudo.ts:15]:
```suggestion
      name: ["-h", "--help"],
```

File: dev/sudo.ts (hunk 2 above)
Comment (cstrnt) [inline dev/sudo.ts:7]:
```suggestion
      name: ["-g", "--group"],
```

File: dev/sudo.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      description: "display help message and exit",
+    },
+    {
+      name: ["-u"],
```
Comment (cstrnt) [inline dev/sudo.ts:19]:
```suggestion
      name: ["-u", "--other-user"],
```

**Review by cstrnt — CHANGES_REQUESTED** [review]
Hey there, thank you very much for contributing! This is a really nice PR which will make our autocompletion even better 😊 I left some comments which are just some minor improvements.

File: dev/sudo.ts (hunk 4 above)
Comment (CastaChick) [reply dev/sudo.ts]: Is it suitable `--user` instead of `--other-user`?
I fixed it by adding `--user`😃

**Comment by CastaChick** [comment]
@cstrnt Thanks for your review!
I've fixed proposed changes 👍

**Review by cstrnt — APPROVED** [review]
Thank you very much for your contribution!

**Review by QuiiBz — APPROVED** [review]
Thanks for your first contribution!

**Pull request merged**